README.md

```diff
@@ -22,4 +22,13 @@ Installation
 
 Documentation
 =============
-Plugin documentation is available [here](doc/Documentation.v0.4.2.md)
\ No newline at end of file
+Plugin documentation is available [here](doc/Documentation.v0.4.2.md)
+
+
+Development
+===========
+Please respect coding rules.
```

Comment:
Maybe we can remove this line, it's a bit harsh and as there is a clang format tool, it's implicit ;)